### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by angelozdev**

### CODE REVIEW

**Comment by AxelDavid45** [comment]
It looks great, I liked it 💪🏼

**Comment by AxelDavid45** [comment]
I'm going to merge it

File: src/__test__/example.test.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
-console.clear()
+describe('example', () => {
+   it('should be true', () => {
+      expect(true).toBe(true)
+   })
+})
 
 /***************************************
  * Preparando el entorno con Jest
  ***************************************/
```
Comment (AxelDavid45) [inline src/__test__/example.test.ts:9]: I'd just changed all your comments from Spanish to English.

**Review by AxelDavid45 — APPROVED** [review]

File: src/__test__/example.test.ts (hunk 1 above)
Comment (angelozdev) [reply src/__test__/example.test.ts:9]: This file is of a jest's course. :sweat_smile:

**Pull request merged**